src/app/modules/item/http-services/join-by-code.service.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { appConfig } from 'src/app/shared/helpers/config';
+import { assertSuccess, SimpleActionResponse } from 'src/app/shared/http-services/action-response';
+
+interface RawGroupInfos {
+  managers: {
+    id: string,
+    login: string,
+    first_name: string|null,
+    last_name: string|null,
+  }[],
+  name: string,
+  require_lock_membership_approval_until: string|null,
+  require_personal_info_access_approval: string|null,
+  require_watch_approval: boolean,
+  root_activity_id: string|null,
+  root_skill_id: string|null,
+}
+
+export interface GroupInfos {
+  managers: {
+    id: string,
+    login: string,
+    firstName: string|null,
+    lastName: string|null,
+  }[],
+  name: string,
+  requireLockMembershipApprovalUntil: string|null,
+  requirePersonalInfoAccessApproval: string|null,
```

Comment:
The goal of the non raw interface is also to convert raw values to a less primitive value. These should be transformed to date (or just removed if not used for now).

Reply:
fixed